package.json

```diff
@@ -1,10 +1,10 @@
 {
-  "name": "generator-az-terra-module",
-  "version": "0.1.4",
-  "description": "Create an Azure terraform module project.",
+  "name": "generator-az-terratest-module",
+  "version": "0.1.0",
+  "description": "Create an Azure terraform module project along with terratest.",
   "author": {
-    "name": "Microsoft Corporation",
-    "url": "https://github.com/Azure/generator-az-terra-module"
+    "name": "foreverXZC",
```

Comment:
Please don't change the author name here.

Reply:
I just used this to publish the package by myself. Will change it back.

Reply:
Please do not do the publish using your own account, this will definitely confuse end user and SEO. Please roll back the publish with your own account.

---
In reply to: [208437684](https://github.com/Azure/generator-az-terra-module/pull/2#discussion_r208437684) [](ancestors = 208437684)

Reply:
If you want to test it locally, you can use npm link.

---
In reply to: [208778322](https://github.com/Azure/generator-az-terra-module/pull/2#discussion_r208778322) [](ancestors = 208778322,208437684)